packages/components/molecules/f-searchbox/src/components/formElements/tests/FormFullAddressSearchModalOverlay.test.js

```diff
@@ -0,0 +1,146 @@
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Vuex from 'vuex';
+import FormFullAddressSearchModalOverlay from '../FormFullAddressSearchModalOverlay.vue';
+
+const localVue = createLocalVue();
+
+localVue.use(Vuex);
+
+const mockState = {
+    isBelowMid: false,
+    isFullAddressSearchEnabled: true,
+    suggestions: ['NG+', 'NG++'],
+    hasInputElevation: false
+};
+
+const mockActions = {
+    setAddress: jest.fn(),
+    getMatchedAreaAddressResults: jest.fn()
+};
+
+const createStore = (state = mockState, actions = mockActions) => new Vuex.Store({
+    modules: {
+        searchbox: {
+            namespaced: true,
+            state,
+            actions
+        }
+    },
+    hasModule: jest.fn(() => true)
+});
+
+describe('`FormFullAddressSearchModalOverlay`', () => {
+    describe('`shouldDisplayModalOverlay`', () => {
+        it('should exist', () => {
+            const wrapper = shallowMount(FormFullAddressSearchModalOverlay, {
+                propsData: {
+                    copy: {}
+                },
+                store: createStore(),
+                localVue
+            });
+
+            expect(wrapper.vm.shouldDisplayModalOverlay).toBeDefined();
+        });
+
+        describe('when invoked', () => {
+            describe('AND the param `value` passed in is `falsy`', () => {
+                it('should `$emit` `ON_FULL_ADDRESS_MODAL_CLOSED` to indicate that the modal should close', () => {
```

Comment:
It feels that `ON_FULL_ADDRESS_MODAL_CLOSED` should be emitted with value `true` to indicate that the modal is closed. Or otherwise be `ON_FULL_ADDRESS_MODAL_OPENED` to be `false`. But maybe I missed something

Reply:
Fixed.